lib/vdsm/storage/lvm.py

```diff
@@ -1384,24 +1384,25 @@ def removeVGbyUUID(vgUUID):
         removeVG(vg.name)
 
 
-def extendVG(vgName, devices, force):
+def extendVG(vgUUID, devices, force):
```

Comment:
This looks wrong. 

We have single caller:

    lib/vdsm/storage/blockSD.py:607: lvm.extendVG(self.sdUUID, devlist, force)

sdUUID is the storage domain UUID, which is the vg name (created by oVirt),
not the vg UUID (created by lvm).

Since this is called with a vg name, the entire change is incorrect.

Reply:
Right, I realized when it was already late last Friday. All started with the methods below that were doing:
```
pvs = listPVNames(vgUUID)
```
But `listPVNames` was expecting a vg_name, not the UUID, so I changed it to translate it from uuid to name before passing it to `listPVNames`. And that triggered other changes like this one. I will revert these lines, since the problem was in the name of the parameter in `def checkVGBlockSizes(vgUUID, vgBlkSize=None):`. It should be either sdUUID or (preferably) vgName, not vgUUID.